File: C++/C++Basic/07.For-Loop-Lab/10.oddEvenSum.cpp
#include <iostream>

using namespace std;

int main()
{
    int count;
    cin >> count;

    int evenSum = 0;
    int oddSum = 0;

    for (int i = 0; i < count; i++)
    {
        int number;
        cin >> number;

        if (i % 2 == 0){
            evenSum += number;
        }
        else
        {
            oddSum += number;
        }
    }
    
    if (evenSum == oddSum)
    {
        cout << "Yes" << endl << "Sum = " << evenSum << endl;
    }
    else
    {
        cout << "No" << endl << "Diff = " << abs(evenSum-oddSum) << endl;
    }

    
    return 0;
}

File: Python/Basic/07.For-Loop-Lab/10.odd_even_sum.py
n = int(input())

odd_sum = 0
even_sum = 0

for idx in range(n):
    number = int(input())

    if idx % 2 == 0:
        even_sum += number
    else:
        odd_sum += number

if even_sum == odd_sum:
    print(f"Yes\nSum = {even_sum}")
else:
    print(f"No\nDiff = {abs(even_sum-odd_sum)}")